api/nodecontrol/client.go

```diff
@@ -186,3 +186,39 @@ func (c *ControlAPIClient) SetLameDuck(nodeId string) (*nodegen.LameduckResponse
 
 	return resp, nil
 }
+
+func (c *ControlAPIClient) MonitorLogs(workloadId, level string) (chan []byte, error) {
```

Comment:
since we know logs and events will have some standard / envelope structure, we should probably attempt to decode that inside the client so that the channels can be of type `*logentry` and `*cloudevent` (just examples).